### PR TITLE
Community page: blue box

### PR DESCRIPTION
**Pull request by dsarkar**
Add blue info box on top of pages
https://www.coronawarn.app/de/community/
https://www.coronawarn.app/en/community/

**DE**
image

**EN**
image

Internal Tracking ID: [EXPOSUREAPP-15177](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-15177)

### CODE REVIEW

File: src/data/community.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
...
...) will no longer be developed as of June 1, 2023. The information on this page is therefore no longer up-to-date. Until April 30, 2023, it was still possible to transmit error reports from the app to the development team. After that, it was possible to report any bugs and errors up until May 31, 2023. However, bug fixes were only created for those bugs that were reported until May 15, 2023. As of May 16, 2023, GitHub repositories began to be archived and it is no longer possible to open new issues. The <a href='https://github.com/corona-warn-app/cwa-website' target='_blank' rel='noopener noreferrer'>repository for the coronawarn.app website</a> is an exception, as the website will be maintained until further notice. The source code of the CWA, as well as the so-called Issues and Pull Requests will remain accessible in the <a href='https://github.com/corona-warn-app' target='_blank' rel='noopener noreferrer'>GitHub repositories</a>. How long this information can be accessed depends on the processes and policies at GitHub. For more information, click here: <a href='../faq/results/#ramp_down' target='_blank'> What is the background for the end of the development of the CWA?</a></div>"
```
Comment (MikeMcC399) [inline src/data/community.json:4]: "so-called" sounds strange. See definition in https://dictionary.cambridge.org/dictionary/english/so-called

I suggest to replace with "associated" which fits better in this sentence, even if it is not a direct translation of the German "sog.".

Comment (dsarkar) [reply src/data/community.json:4]: @MikeMcC399 many thanks for your suggestion. Will replace as suggested.

**Review by Ein-Tim — APPROVED** [review]
Text is apporved!

**Comment by MikeMcC399** [comment]
Good morning @dsarkar!

The deployment failed.

**Comment by Ein-Tim** [comment]
See https://github.com/corona-warn-app/cwa-website/actions/runs/5138188257

**Pull request merged**